yoti-stub/src/services/YotiRequestProcessor.ts

```diff
@@ -298,4 +323,42 @@ export class YotiRequestProcessor {
 		}
 
 	}
+
+	/***
+	 * GET /sessions/{sessionId}/media/{mediaId}/content
+	 */
+	async getMediaContent(mediaId: string): Promise<Response> {
+
+		const lastUuidChars = mediaId.slice(-4);
+		this.logger.info({ message: "last 4 ID chars", lastUuidChars});
+
+		switch(lastUuidChars) {
+			case '0000': // UK Passport
```

Comment:
please update the mapping for these in https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3537600538/F2F+Yoti+Stub+Design+Scenarios